ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.html

```diff
@@ -0,0 +1,208 @@
+                    (embeddedComponentsValidationStatus)="updateEmbeddedComponentValidationStatus(1, $event)"
+                    (embeddedComponentBusy)="embeddedComponentBusy$[1].next($event)">
+                </ddp-activity-section>
+
+                <!-- closing section -->
+                <ng-container *ngIf="model.closing">
+                    <ddp-activity-section
+                        [section]="model.closing"
+                        [readonly]="isReadonly() || dataEntryDisabled"
+                        [validationRequested]="validationRequested"
+                        [studyGuid]="studyGuid"
+                        [activityGuid]="activityGuid"
+                        (visibilityChanged)="updateVisibility($event)"
+                        (embeddedComponentsValidationStatus)="updateEmbeddedComponentValidationStatus(2, $event)"
+                        (embeddedComponentBusy)="embeddedComponentBusy$[2].next($event)">
+                    </ddp-activity-section>
+                </ng-container>
+
+                <ng-container *ngIf="shouldShowReadonlyHint">
+                    <div class="infobox" [innerHTML]="model.readonlyHint"></div>
+                </ng-container>
+
+                <ng-container *ngIf="model.lastUpdatedText">
+                    <span class="last-updated">{{model.lastUpdatedText}} </span>
+                </ng-container>
+
+                <div class="activity-buttons"
+                     [ngClass]="{'activity-buttons_mobile': (!isStepped || isLastStep) && isAgree() && isLoaded && !isReadonly()}">
+                    <ng-container *ngIf="isLoaded && isStepped; then navigationButtons"></ng-container>
+                    <ng-container *ngIf="(!isStepped || isLastStep) && isLoaded; then lastStepButtons"></ng-container>
+                    <ng-container
+                        *ngIf="(!isStepped || isLastStep) && isAgree() && isLoaded && !isReadonly(); then agreementButtons">
+                    </ng-container>
+                </div>
+
+                <div *ngIf="displayGlobalError$ | async" class="ErrorMessage">
+                    <span translate>SDK.ValidateError</span>
+                </div>
+                <div *ngIf="communicationErrorOccurred" class="ErrorMessage">
+                    <span translate>SDK.CommunicationError</span>
+                </div>
+            </ng-container>
+        </div>
+    </section>
+</main>
+
+<!--------------------------Used templates------------------------------>
+<ng-template #spinner>
+    <section class="section section-spinner">
+        <mat-spinner></mat-spinner>
+    </section>
+</ng-template>
+
+<ng-template #stepsWithCircle>
+    <div class="activity-steps" [ngClass]="{'vertical': isVerticalProgress}">
+        <ng-container *ngFor="let section of model.sections; index as i; last as isLastStep">
+            <ng-container *ngIf="section.visible">
+                <div class="activity-step" cdkMonitorElementFocus
```

Comment:
**_cdkMonitorElementFocus_** is used there as to catch a source of focus event on this element (an indicator step) and style it with outline for Tab key navigation case.